src/main/java/com/patchworkmc/Patchwork.java

```diff
@@ -153,21 +150,15 @@ private ForgeModJar parseModManifest(Path jarPath) throws IOException, URISyntax
 		URI inputJar = new URI("jar:" + jarPath.toUri());
 
 		FileConfig toml;
-		AccessTransformerList accessTransformers = null;
-
+		ForgeAccessTransformer at;
 		try (FileSystem fs = FileSystems.newFileSystem(inputJar, Collections.emptyMap())) {
 			Path manifestPath = fs.getPath("/META-INF/mods.toml");
 			toml = FileConfig.of(manifestPath);
 			toml.load();
-
 			try {
-				accessTransformers = AccessTransformerList.parse(fs.getPath("/META-INF/accesstransformer.cfg"));
-			} catch (Exception e) {
-				LOGGER.throwing(Level.ERROR, new RuntimeException("Unable to parse access transformer list", e));
-			}
-
-			if (accessTransformers == null) {
-				accessTransformers = new AccessTransformerList(new ArrayList<>());
+				at = ForgeAccessTransformer.parse(fs.getPath("/META-INF/accesstransformer.cfg"));
+			} catch (ManifestParseException ex) {
+				at = null;
```

Comment:
The exception shouldn't be silently swallowed here, there should be better error handling / checking (such as whether it exists at all)